backend/trivia-server.js

```diff
@@ -0,0 +1,135 @@
+import { update } from '@reshuffle/db';
+import errorType from './constants';
+
+/**
+ * Get trivia questions
+ */
+/* @expose */
+export async function getTriviaData() {
+  try {
+    return update('trivia', (result = data) => {
+      return result;
+    });
+  } catch (error) {
+    console.error('getTriviaData -> error', error.message);
+    return new Error(errorType.fetch);
+  }
+}
+
+/**
+ * Initial data for trivia questions
+ */
+const data = [
+  {
+    category: 'React 101',
+    questions: [
+      {
+        points: 100,
+        question:
+          '<p>What part of your application does React focus on?</p><ol><li>The Model</li><li>The View</li><li>The Controller</li></ol>',
+        answer: 'B. The View',
+      },
+      {
+        points: 200,
+        question: '_____ lets you create JavaScript objects using HTML syntax',
+        answer: 'JSX',
+      },
+      {
+        points: 300,
+        question: 'What tool can you use to transpile JSX?',
+        answer: 'Babel',
+      },
+      {
+        points: 400,
+        question:
+          'React uses a _____________ to limit direct manipulation of the DOM and improve performance',
+        answer: 'Virtual DOM',
+      },
+    ],
+  },
+  {
+    category: 'React 201',
+    questions: [
+      {
+        points: 100,
+        question:
+          "<p>Where can you define a component's initial state when you use the ES6 Class syntax?</p><ol><li>getInitialState()</li><li>getInitialProps()</li><li>The component's constructor</li></ol>",
+        answer: "C. The component's constructor",
+      },
+      {
+        points: 200,
+        question:
+          "Using npm, which package should you require/import in addition to 'react' to render a React component in an existing DOM element of your HTML file?",
+        answer: 'react-dom',
+      },
+      {
+        points: 300,
+        question:
+          '<p>Which lifecycle function should you use to set default property values?</p><ol><li>getInitialState</li><li>getInitialProps</li><li>getDefaultProps</li></ol>',
+        answer: 'C. getDefaultProps',
+      },
+      {
+        points: 400,
+        question:
+          'Which lifecycle method is invoked once, immediately after the initial rendering occurs?',
+        answer: 'componentDidMount',
```

Comment:
Some answers/questions have parentheses after functions like `getInitialProps()` but this one does not